Offseason2022/src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -308,6 +309,7 @@ private void initSmartDashboard( )
     SmartDashboard.putNumber("DTL_throttlePidKd", m_throttlePidKd);
     SmartDashboard.putNumber("DTL_throttleMax", m_throttleMax);
     SmartDashboard.putNumber("DTL_throttleShape", m_throttleShape);
+    SmartDashboard.putBoolean("DT_throttleZeroed", m_throttleZeroed);

```

Comment:
Just bump this up to near DT_stopTolerance, since this group (DTL_) are all drivetrain limelight values.